baseapp-chats/baseapp_chats/permissions.py

```
@@ -3,45 +3,82 @@
+            current_profile = obj["profile"]
+
+            if not getattr(room, "is_group", False):
+                return False
+
+            chat_room_participant = ChatRoomParticipant.objects.filter(
+                room=room, profile_id=current_profile.pk
+            ).first()
+            if (
+                not chat_room_participant
+                or not chat_room_participant.role
+                == ChatRoomParticipant.ChatRoomParticipantRoles.ADMIN
```

Ruff (0.8.2)

Use `chat_room_participant.role != ChatRoomParticipant.ChatRoomParticipantRoles.ADMIN` instead of `not chat_room_participant.role == ChatRoomParticipant.ChatRoomParticipantRoles.ADMIN`

Replace with `!=` operator

(SIM201)